app/javascript/packs/navigation.js

```diff
@@ -2,7 +2,46 @@ import { toggleNavigation, hideAllToggleable } from '../src/Navigation/utils/tog
 import { ContextSelectorWrapper } from '../src/Navigation/components/ContextSelector'
 
 document.addEventListener('DOMContentLoaded', () => {
-  window.toggleNavigation = toggleNavigation
-  window.hideAllToggleable = hideAllToggleable
-  window.ContextSelector = ContextSelectorWrapper
+  const apiSelector = 'api_selector'
+  const apiSelectorNode = document.getElementById(apiSelector)
+  const apiSelectorData = JSON.parse(apiSelectorNode.getAttribute('data'))
```

Comment:
it is only `data`? I thought HTML 5 data attribute should be prefixed by `data-`

Reply:
It's passed from the view as single data attr: https://github.com/3scale/porta/pull/387/files#diff-c831c6b813f4dcc54721733f25630f1cR1

Reply:
That is not the point. It should be something like `data-something`